src/hevm/src/EVM/Format.hs

```diff
@@ -163,37 +199,71 @@ showTrace dapp trace =
     fullAbiMap = view dappAbiMap dapp
   in case view traceData trace of
     EventTrace (Log _ bytes topics) ->
-      case topics of
-        [] ->
-          mconcat
+      let logn = mconcat
+            [ "\x1b[36m"
+            , "log" <> (pack (show (length topics))) <> "("
+            , intercalate ", " (map (pack . show) topics)
```

Comment:
is there a `,` missing between the topic and the contents here?

Reply:
good catch, yes